src/pykeen/hpo/hpo.py

```diff
@@ -604,52 +603,52 @@ def hpo_pipeline(
     )
 
     # 2. Model
-    model: Type[Model] = get_model_cls(model)
-    study.set_user_attr('model', normalize_string(model.__name__))
-    logger.info(f'Using model: {model}')
+    model_cls: Type[Model] = model_resolver.lookup(model)
+    study.set_user_attr('model', normalize_string(model_cls.__name__))
+    logger.info(f'Using model: {model_cls}')
     # 3. Loss
-    loss: Type[Loss] = model.loss_default if loss is None else get_loss_cls(loss)
-    study.set_user_attr('loss', normalize_string(loss.__name__, suffix=_LOSS_SUFFIX))
-    logger.info(f'Using loss: {loss}')
+    loss_cls: Type[Loss] = model_cls.loss_default if loss is None else loss_resolver.lookup(loss)
+    study.set_user_attr('loss', normalize_string(loss_cls.__name__, suffix=loss_resolver.suffix))
```

Comment:
Make `normalize_string(loss_cls.__name__, suffix=loss_resolver.suffix)` a method of the resolver? This should be the "inverse" resolution, correct?

Reply:
Hmm good point